packages/accounts/src/light-account/account.ts

```diff
@@ -1,25 +1,35 @@
 import {
   SimpleSmartContractAccount,
-  SmartAccountProvider,
+  type ISmartAccountProvider,
   type SignTypedDataParams,
   type SmartAccountSigner,
 } from "@alchemy/aa-core";
 import {
   concatHex,
   decodeFunctionResult,
   encodeFunctionData,
+  fromHex,
+  trim,
   type Address,
+  type Chain,
   type FallbackTransport,
   type Hash,
   type Hex,
   type Transport,
 } from "viem";
+import { createMultiOwnerMSCA } from "../msca/multi-owner-account.js";
+import { getDefaultMSCAFactoryAddress } from "../msca/utils.js";
 import { LightAccountAbi } from "./abis/LightAccountAbi.js";
 import { LightAccountFactoryAbi } from "./abis/LightAccountFactoryAbi.js";
 
 export class LightSmartContractAccount<
   TTransport extends Transport | FallbackTransport = Transport
 > extends SimpleSmartContractAccount<TTransport> {
+  static readonly implementationAddress =
```

Comment:
nit: this should be configurable and tied to the response of `getDefaultFactoryAddress`